code/datums/gamemode/role/vampire_role.dm

```diff
@@ -160,6 +162,10 @@
 		to_chat(M, "<span class='warning'>Remove their mask!</span>")
 		return FALSE
 
+	if(vampire_teeth.amount == 0)
```

Comment:
Is `vampire_teeth` guaranteed not to be null here? What about frankensteins, I remember them being weird with butchering drops?